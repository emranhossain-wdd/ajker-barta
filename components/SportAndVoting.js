import React from 'react';
import Sports from './Home/Sports/Sports';
import Voting from './Voting/Voting';

const SportAndVoting = () => {
    return (
        <div className='container mx-auto lg:pb-5 py-10 border-b  border-gray-200'>
        <div className="grid grid-cols-3 gap-2">
 
  <div className="col-span-2 ..."><Sports></Sports></div>
  <div className="..."><Voting></Voting></div>
  
</div>
</div>
    );
};

export default SportAndVoting;