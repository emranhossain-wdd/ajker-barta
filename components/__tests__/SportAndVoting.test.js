import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import SportAndVoting from '../SportAndVoting'

it('render SportAndVoting component without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<SportAndVoting />, div)
})
it('maching snapshot', () => {
  const tree = renderer.create(<SportAndVoting />)
  expect(tree).toMatchSnapshot()
})