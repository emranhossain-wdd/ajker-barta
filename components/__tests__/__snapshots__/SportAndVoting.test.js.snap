// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maching snapshot 1`] = `
<div
  className="container mx-auto lg:pb-5 py-10 border-b  border-gray-200"
>
  <div
    className="grid grid-cols-3 gap-2"
  >
    <div
      className="col-span-2 ..."
    >
      <div
        className="container"
      >
        <h2
          className="text-xl font-medium mt-5 mb-2"
        >
          <span
            className="text-blue-900"
          >
            <a
              href="/international"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Sports
            </a>
             
          </span>
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium text-red-500 css-i4bv87-MuiSvgIcon-root"
            data-testid="ChevronRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
        </h2>
        <div
          className="grid grid-cols-12 gap-4"
        >
          <div
            className="col-span-12 md:col-span-6 mb-3 lg:mb-0"
          >
            <div
              className="p-3 rounded-md border-r border-b"
            />
          </div>
          <div
            className="col-span-12 md:col-span-6"
          >
            <div
              className=""
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="..."
    >
      <div
        className="container"
      >
        <h2>
          this is Voting
        </h2>
        <div
          className="flex gap-6"
        />
      </div>
    </div>
  </div>
</div>
`;
